src/esp/sim/Simulator.h

```diff
@@ -591,6 +591,22 @@ class Simulator {
    */
   bool isNavMeshVisualizationActive();
 
+  /**
+   * @brief Compute a trajectory visualization for the passed points.
+   * @param assetName The name to use for the asset
+   * @param pts The points of a trajectory, in order
+   * @param numSegments The number of the segments around the circumference of
+   * the tube. Must be greater than or equal to 3.
+   * @param numInterp The number of interpolations between each trajectory point
+   * @param radius The radius of the tube.
+   * @return The ID of the object created for the visualization
+   */
+  int showTrajectoryVisualization(const std::string& assetName,
```

Comment:
Can you explain what you mean by asset here?

Asset is a confusing term for me. I think of assets as persistent things (persist across multiple runs of the program). Things on disk or other storage that get loaded into memory.